adapters/repos/db/migrator.go

```diff
@@ -54,7 +54,18 @@ func (m *Migrator) AddClass(ctx context.Context, kind kind.Kind, class *models.C
 }
 
 func (m *Migrator) DropClass(ctx context.Context, kind kind.Kind, className string) error {
-	return fmt.Errorf("dropping a class not (yet) supported")
+	idx := m.db.GetIndex(kind, schema.ClassName(className))
+	if idx == nil {
+		return errors.Errorf("exist index '%s'", idx.ID())
```

Comment:
Should we allow deleting of a non-existing index, so that deletes can be done idempotently without having to check if what should be deleted exists first? Or is there anything at the `usecase` layer that would already prevent this?

Reply:
I will make sure that our drop operation is idempotent, I'll also remove this part of code, as all of this will be done in `DeleteIndex()` method